resources/definitions/zonestar_z5xm2_printer.def.json

```diff
@@ -0,0 +1,41 @@
+{
+    "name": "ZONESTAR Z5XM2",
```

Comment:
I can't find any information about this printer. Can you provide a link to a spec sheet or purchasing page somewhere so that we can check this definition file?